### PR TITLE
DPL-937 - Bait capture library prep pipeline

### LINKED ISSUE #1406: DPL-937 - As an R&D scientist I would like to create a new pipeline for the in-house Bait Capture library preparation pipeline for RVI
**User story**
As an R&D scientist I would like to create a new pipeline for the in-house Bait Capture library (BCL) preparation pipeline for RVI. A new pipeline is needed as this is more appropriate for reporting and maintenance especially as this information will need to be disseminated to external collaborators.

**Who are the primary contacts for this story**
Di / Scott / Steve

**Who is the nominated tester for UAT**
Di / Scott

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Lab to be able to run BCL pipeline distinct from RNA PolyA (RNAA) library prep, LCMB pipeline, ISC (Twist pulldown) and other existing pipelines
- [ ] NPG contacted to discuss creation of BCL specific library type to use the 'standard' analysis route
- [ ] New Integration suite test for BCL pipeline to run cleanly

**Dependencies**
This story is blocked by the following dependencies:
- sanger/sequencescape#3897
- sanger/sequencescape#3895
- sanger/sequencescape#3902

**Additional Context**

[Lucid Chart showing LIMS steps](https://lucid.app/lucidchart/c1a5c6a6-f4d4-4cfe-b216-e5a8e9dc34ce/edit?viewport_loc=-1348%2C-774%2C2367%2C1395%2C0_0&invitationId=inv_14681694-a000-47a8-bca7-104cf2537d80)

References

- sanger/general-backlog-items#334

### PR DESCRIPTION
**Pull request by BenTopping**
Closes #1406

#### Changes proposed in this pull request

- Adds plate purposes and pipeline

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

File: config/robots.rb [hunk 1]
```diff
@@ -3061,4 +3061,232 @@
         }
       }
     )
+
```
Comment (andrewsparkes) [inline config/robots.rb:3064]: these all look good to me
we just need to check with them if they want us to verify the robot barcodes

File: spec/controllers/tube_creation_controller_spec.rb [hunk 2]
```diff
@@ -43,7 +43,7 @@
         it 'creates a tube' do
           get :new,
               params: {
-                limber_tube_id: parent_uuid,
+                limber_plate_id: parent_uuid,
```
Comment (andrewsparkes) [inline spec/controllers/tube_creation_controller_spec.rb:46]: not following, why have these 3 been reversed? was this a bug?

**Review by andrewsparkes — COMMENTED** [review]
looks good, just confused about the changes in that tube creation controller

File: spec/controllers/tube_creation_controller_spec.rb (hunk 2 above)
Comment (BenTopping) [reply spec/controllers/tube_creation_controller_spec.rb:46]: Yeah I believe the logic was incorrect here. 'From a plate parent create a tube' So the parent_uuid should be coming from a plate.

**Pull request merged**